src/components/LiveGraph.tsx

```diff
@@ -85,30 +84,21 @@ const LiveGraph = () => {
     }
   }, [chart, isConnected, status]);
 
-  // Draw on graph to display that users are recording
-  // Ideally we'd do this without timing the recording again!
-  const [isTimingRecording, setIsTimingRecording] = useState<boolean>(false);
-  const dataWindow = useStore((s) => s.dataWindow);
+  // Draw on graph to display that users are recording.
   const isRecording = useStore((s) => s.isRecording);
   useEffect(() => {
-    if (isRecording && !isTimingRecording) {
-      {
-        // Set the start recording line
-        const now = new Date().getTime();
-        recordLines.append(now - 1, -2, false);
-        recordLines.append(now, 2.3, false);
-        setIsTimingRecording(true);
-      }
-
-      setTimeout(() => {
-        // Set the end recording line
-        const now = new Date().getTime();
-        recordLines.append(now - 1, 2.3, false);
-        recordLines.append(now, -2, false);
-        setIsTimingRecording(false);
-      }, dataWindow.duration);
+    if (isRecording) {
+      // Set the start recording line
+      const now = new Date().getTime();
```

Comment:
I'd feel better about this if this could get the timestamps from the code that controls the recording.
Maybe we have a way to listen for recording events that include timestamps rather than using React state.

```
const recorder = useRecorder()
useEffect(() => {
  // With clean-up omitted, events carry a timestamp
  recorder.addEventListener("recordingstart", ...)
  recorder.addEventListener("recordingstop", ...)
});
```

We can look at this in a follow up PR.